vendor/github.com/containerd/containerd/vendor.conf

```diff
@@ -57,22 +58,26 @@ github.com/golang/glog 44145f04b68cf362d9c4df2182967c2275eaefed
 github.com/google/gofuzz 44d81051d367757e1c7c6a5a86423ece9afcf63c
 github.com/hashicorp/errwrap 7554cd9344cec97297fa6649b055a8c98c2a1e55
 github.com/hashicorp/go-multierror ed905158d87462226a13fe39ddf685ea65f1c11f
-github.com/json-iterator/go 1.0.4
-github.com/opencontainers/runtime-tools 6073aff4ac61897f75895123f7e24135204a404d
+github.com/json-iterator/go f2b4162afba35581b6d4a50d3b8f34e33c144682
+github.com/modern-go/reflect2 05fbef0ca5da472bbf96c9322b84a53edc03c9fd
```

Comment:
Same here; this is somewhere between 1.0.0 and 1.0.1 https://github.com/modern-go/reflect2/compare/05fbef0ca5da472bbf96c9322b84a53edc03c9fd...1.0.1 - I can open a PR upstream for that